plugins/qtosgrave/osgviewerwidget.cpp

```diff
@@ -1452,36 +1500,46 @@ void QOSGViewerWidget::SetCurrentCameraManipulator(osgGA::CameraManipulator* man
     _osgview->setCameraManipulator(manipulator);
 }
 
-double QOSGViewerWidget::GetCurrentManipulatorDistanceToFocus()
+double QOSGViewerWidget::GetCameraDistanceToFocus()
 {
     osgGA::OrbitManipulator* currentManip = dynamic_cast<osgGA::OrbitManipulator*>(GetCurrentCameraManipulator().get());
     return currentManip->getDistance();
-
 }
 
-void QOSGViewerWidget::SetCurrentManipulatorDistanceToFocus(double distance)
+void QOSGViewerWidget::SetCameraDistanceToFocus(double distance)
 {
     if( distance <= 0 ) {
         return;
     }
+
+    if(IsInOrthoMode()) {
+        RAVELOG_WARN("performing hard SetCameraDistanceToFocus when in ortho mode, please use Zoom() to change zoom independently from the current view mode.");
```

Comment:
Should have early return after warning message

Reply:
So maybe an error is better? a warning suggest that its still ok to do but not adisable... I think..

Reply:
I think warn is also ok